linera-service/src/cli_wrappers/local_kubernetes_net.rs

```diff
@@ -1,11 +1,12 @@
 // Copyright (c) Zefchain Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-use super::{
-    docker::DockerImage, kind::KindCluster, kubectl::KubectlInstance, util::get_github_root,
-};
 use crate::{
-    cli_wrappers::{helmfile::HelmFile, ClientWrapper, LineraNet, LineraNetConfig, Network},
+    cli_wrappers::{
+        docker::DockerImage, helmfile::HelmFile, kind::KindCluster, kubectl::KubectlInstance,
+        util::get_github_root, wallet::FaucetOption, ClientWrapper, LineraNet, LineraNetConfig,
```

Comment:
I think `wallet::FaucetOption` needs to be under `#[cfg(any(test, feature = "test"))]`

Reply:
right!